test/Sentry.Unity.Editor.iOS.Tests/NativeOptionsTests.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using NUnit.Framework;
+
+namespace Sentry.Unity.Editor.iOS.Tests
+{
+    public class NativeOptionsTests
+    {
+        [Test]
+        public void GenerateOptions_NewSentryOptions_Compiles()
+        {
+            if (Environment.OSVersion.Platform != PlatformID.MacOSX)
+            {
+                Assert.Inconclusive("Skipping: Non MacOSX");
```

Comment:
OSX is the old name, what about:

```suggestion
                Assert.Inconclusive("Skipping: Not on macOS");
```